### DISCUSSION TITLE
更新v1.1.0后，每天被服务器踹下线一次

### DISCUSSION THREAD

**Discussion by PercyDan54** [Mrs4s/go-cqhttp#2257] [Q&A]
协议是安卓手表扫码登录，其他协议都是登录问滑块，滑完了就是45。
已经持续两天被服务器凌晨踹下线，手机端也会被下线
签名服务器：https://qsign.loli.vet/
![Y`1QYM8G~ J)RJQGSL )JB9](https://github.com/Mrs4s/go-cqhttp/assets/50285552/462be122-697d-421d-b881-e4152604a33e)

**Comment by zhzhongshi** [comment]
同

**Reply by zhzhongshi** [reply]
每次踹下去都得怼人脸

**Comment by PercyDan54** [comment]
![image](https://github.com/Mrs4s/go-cqhttp/assets/50285552/6a63d09d-383c-411d-95db-a1ceec73019d)

发送消息依然风控

**Comment by bywhite0** [comment]
建议先自行搭建签名服务器，可能是使用该服务器的用户过多（

**Comment by best4566** [comment]
good job friand
